server/services/apollo/src/index.js

```diff
@@ -170,4 +183,28 @@ async function runServerForever() {
   }
 }
 
+async function send_telemetry_event(event) {
+  if (TELEMETRY_ENABLED) {
+    try {
+      // TODO add timeout
+      fetch('https://sens-o-matic.prefect.io/', {
+        method: 'post',
+        body: JSON.stringify({
+          source: 'prefect_server',
+          type: event,
+          payload: { id: TELEMETRY_ID }
+        }),
+        headers: {
+          'Content-Type': 'application/json',
+          'X-Prefect-Event': 'prefect_server-0.0.1'
+        }
+      })
+        .then(res => res.json())
+        .then(json => log(json))
+    } catch (error) {
+      log(error)
```

Comment:
Should probably add some context around the log message. I'm not super familiar with javascript, but perhaps this works?

```suggestion
      log("Error sending telemetry event: ${error.message}")
```

Reply:
I agree, will add